crates/fixt/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ---
-default_semver_increment_mode: !pre_patch beta-rc
+default_semver_increment_mode: patch
```

Comment:
```suggestion
default_semver_increment_mode: !pre_patch beta-dev
semver_increment_mode: patch
```

Reply:
this will cause just the next release to be a patch release and all subsequent releases be beta-dev releases until configured otherwise.

Reply:
(i suggest applying this to all crates)

Reply:
Done